openedx_events/content_authoring/data.py

```diff
@@ -221,7 +221,10 @@ class LibraryCollectionData:
     Arguments:
         library_key (LibraryLocatorV2): a key that represents a Blockstore-based content library.
         collection_key (str): identifies the collection within the library's learning package
+        lazy (bool): indicate whether the sender doesn't want to wait for handler to finish execution,
+        i.e., the handler can run the task in background. By default it is False.
```

Comment:
Please, don't consider this comment a blocker. I’m having a bit of trouble understanding what lazy means in this context without reading the description. Do you imagine other cases where this field is `True` besides bulk updates? Should the name suggest that?

Reply:
@mariajgrimaldi We want to control the way handler executes some functions (search indexing in this case). How about renaming this field to `async` or `background`. 
cc: @rpenido

Reply:
Yes. That’d be more self-explanatory. Thank you!

Reply:
Done.